src/content/docs/new-relic-solutions/observability-maturity/operational-efficiency/dg-optimizing.mdx

```diff
@@ -619,13 +633,12 @@ The different approaches:
           * Logs generated per cluster
         </Callout>
 
-        It's not surprising that a complex and decentralized system like Kubernetes has the potential to generate a lot of telemetry fast.  There are a few good approaches to managing data ingest in Kubernetes.  These will be very straightforward if you are using observability as code in your K8s deployments.
+        It's not surprising that a complex and decentralized system like Kubernetes has the potential to generate a lot of telemetry fast.  There are a few good approaches to managing data ingest in Kubernetes.  These will be very straightforward if you are using observability as code in your K8s deployments.  We highlly recommend you install this `Kubernetes Data Ingest Analysis` dashboard for making any decisions about reducing K8s ingest.  The dashbaord is available in this [quickstart](https://newrelic.com/instant-observability/infrastructure-integrations-data-analysis/8e31a0ae-81c0-4df0-a119-a0ada9ec16fa).
```

Comment:
@JimHagan, you can remove the extra space before "We."There's an extra "L" in "highly," but I'm not sure you even need that word.